release-channels-and-staged-deployments/README.md

```diff
@@ -0,0 +1,379 @@
+# Release channels and staged deployments
+
+The document talks about releases, deployments, how we deploy components in different ways, when we deploy them.
+
+It starts with an overview of what we have today, and then continues with how we can structure releases an deployments in the future.
+
+## 1. What and when we release and deploy today?
+
+### 1.1. What are we deploying?
+
+1. cluster-x app
+  - Cluster resources
+  - Most critical apps: CPI, CNI, CSI
+  - Owners: LM, CI, OPI
```

Comment:
I am not familiar with these acronyms, can you link to Glosary or explain in the first appearance?